desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraphTransformers.java

```diff
@@ -235,7 +236,16 @@ public Icon transform(Object obj) {
                 } else {
                     configured = componentBuilder.isConfigured(false);
                 }
-                return IconUtils.getDescriptorIcon(descriptor, configured, IconUtils.ICON_SIZE_LARGE);
+
+                Icon descriptorIcon = IconUtils.getDescriptorIcon(descriptor, configured, IconUtils.ICON_SIZE_LARGE);
+
+                if (descriptor instanceof RemoteTransformerDescriptor) {
```

Comment:
Hmm maybe this should be a part of the above ```IconUtils.getDescriptorIcon``` method call? Then we would benefit from the overlay also in other places where the icon might be rendered. And it could also be correleated with the "configured" flag (which may already be false - yielding a 'double' overlay).